AsyncDisplayKit/ASCollectionView.h

```diff
@@ -192,6 +192,16 @@
 - (void)reloadItemsAtIndexPaths:(NSArray *)indexPaths;
 
 /**
+ * Relayouts the specified item.
+ *
+ * @param indexPath The index path identifying the item to relayout.
+ *
+ * @discussion This method must be called from the main thread. The relayout is excuted on main thread.
```

Comment:
You should probably mention here that it does perform a re-measurement, with the same constrained size (the width or height of the view in the direction orthogonal to scrolling, plus CGFLOAT_MAX).

If you could also explain whether this does work when called inside of an animation block (to animate all of the changes), I think many people reading the comments will be trying to do that.  Really clean implementation!